conf/application-json-logger.xml

```diff
@@ -5,7 +5,7 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="uk.gov" level="DEBUG"/>
+    <logger name="uk.gov" level="INFO"/>
```

Comment:
Can this please be changed to `<logger name="uk.gov" level="${logger.uk.gov:-INFO}"/>`?
It will allow us to override without changing the code.

Reply:
👍